### PR TITLE
Add user role to `/auth/whoami` response

### PR DESCRIPTION
**Pull request by haoming29**
Add user role ("admin" for user with admin privilege or "user" if they don't) so that UI can do RBAC.

### CODE REVIEW

**Comment by haoming29** [comment]
@CannonLock This PR should give you what you need

File: web_ui/authentication.go [hunk 1]
```diff
@@ -274,6 +324,27 @@ func resetLoginHandler(ctx *gin.Context) {
 	}
 }
 
+func whoamiHandler(ctx *gin.Context) {
```
Comment (joereuss12) [inline web_ui/authentication.go:327]: Maybe add comments before this function as well, just to remain consistent even though it is pretty self-explanitory

**Review by joereuss12 — CHANGES_REQUESTED** [review]
LGTM other than this one simple change

**Review by joereuss12 — APPROVED** [review]
LGTM

**Pull request merged**